h5io/tests/test_io.py

```diff
@@ -38,6 +38,27 @@ def test_hdf5():
                   overwrite=True)
     assert_raises(TypeError, write_hdf5, test_file, object, overwrite=True)
 
+    write_hdf5(test_file, 1, title='first')
+    write_hdf5(test_file, 2, title='second')
```

Comment:
No, you have to explicit overwrite = True, the default would have raised an error. I will go back to the default raise error behaviour and make overwrite = 'update' to append (or replace value if title already exists)